src/WebSdk/Publish/Targets/Microsoft.NET.Sdk.Publish.props

```diff
@@ -15,7 +15,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <EnableSdkContainerSupport Condition="'$(EnableSdkContainerSupport)' == ''">true</EnableSdkContainerSupport>
   </PropertyGroup>
 
+  <!-- Container publishing is only supported for a single TFM at a time at this point, so only import those targets in
+       'inner' builds  -->
   <Import Project="$(MSBuildThisFileDirectory)..\..\..\Containers\build\Microsoft.NET.Build.Containers.props"
-          Condition="'$(EnableSdkContainerSupport)' == 'true' AND Exists('$(MSBuildThisFileDirectory)..\..\..\Containers\build\Microsoft.NET.Build.Containers.props')" />
+          Condition="'$(EnableSdkContainerSupport)' == 'true'
+                     AND Exists('$(MSBuildThisFileDirectory)..\..\..\Containers\build\Microsoft.NET.Build.Containers.props')
+                     AND '$(TargetFramework)' != '' " />
```

Comment:
Is this true at this point for a single-targeted project? it's usually in project body and .props files are often before that.

Reply:
you're right - I got over-zealous. There aren't any dangerous properties set in this props file anyway so it should be safe to always import.